msgraph-developer-proxy/appsettings.json

```diff
@@ -1,4 +1,36 @@
 {
+  "handlers": [
+    {
+      "pluginPath": "GraphProxyPlugins\\msgraph-proxy-plugins.dll",
+      "name": "SelectGuidancePlugin"
+    },
+    {
+      "pluginPath": "GraphProxyPlugins\\msgraph-proxy-plugins.dll",
+      "name": "SdkGuidancePlugin"
+    },
+    {
+      "configSection": "mocksPlugin",
+      "pluginPath": "GraphProxyPlugins\\msgraph-proxy-plugins.dll",
+      "name": "MockResponsePlugin",
+      "disabled": false
+    },
+    {
+      "configSection": "randomErrorsPlugin",
+      "pluginPath": "GraphProxyPlugins\\msgraph-proxy-plugins.dll",
+      "name": "RandomErrorPlugin",
+      "urlsToWatch": [
+        "https://graph.microsoft.com/v1.0/*",
```

Comment:
Shouldn't we allow testing all URLs with random errors?

Reply:
Yes, we should, I wanted to have an example of a plugin level set of urlsToWatch for development and testing, I'll move this to the mocksPlugin as we only have graph based mocks in our default set.
Actually I'll disable that plugin in the default set up too.